cves/CVE-2013-7014.yml

```
@@ -342,7 +350,11 @@ mistakes:
     Use those questions to inspire your answer. Don't feel obligated to answer
     every one. Write a thoughtful entry here that those ing the software
     engineering industry would find interesting.
-  answer:
+  answer: |
+    The biggest mistake that caused this vulenrability was that they were trying to write the
```

Typo of "vulnerability".
```suggestion
    The biggest mistake that caused this vulnerability" was that they were trying to write the
```